### PR TITLE
Feature/register auth0 callback url

### PR DESCRIPTION
**Pull request by BenCoffeed**
- [x] Add the Clubhouse Story ID: [ch21300]
- [ ] Add QA tasks

### CODE REVIEW

File: src/assets/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "domain": "roundingwell-care-team.auth0.com",
-  "client_id": "zmfrKoBuMKsm5H4ZekNHeA4FRzIS44cz",
+  "domain": "rwell-careops-dev.auth0.com",
```
Comment (paulfalgout) [inline src/assets/config.json]: we can't change this.  It's the config for local development.  We'll need a different `config.json` for the `*.preview.`s

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,6 +52,27 @@ jobs:
...
+      - run:
+          name: install curl
+          command: apk --no-cache add curl
+      - run:
+          name: Update GitHub status
+          command: |
+            echo "export VERSION_SUFFIX=$(echo "$CIRCLE_BRANCH" | tr '[:punct:][:blank:]' '-' | tr '[:upper:]' '[:lower:]')" >> $BASH_ENV
+            source $BASH_ENV  
+            source .circleci/bin/auth0_tools
+            add_auth0_callback "https://fe-${VersionSuffix}.preview.roundingwell.com/authenticated"
```
Comment (paulfalgout) [inline .circleci/config.yml]: This will also need to happen for Allowed Web Origins and Logout URLs

File: src/assets/config.json (hunk 1 above)
Comment (BenCoffeed) [reply src/assets/config.json]: 👍

File: .circleci/config.yml (hunk 2 above)
Comment (BenCoffeed) [reply .circleci/config.yml]: Taken care of.

File: src/assets/config.json (hunk 1 above)
Comment (BenCoffeed) [reply src/assets/config.json]: Fixed.

**Comment by BenCoffeed** [comment]
This is now operating as expected. Still want to hold off until we get the config.json redirector deployed.

**Pull request merged**